README.md

```diff
@@ -1,2 +1,63 @@
-# fxa-geodb
```

Comment:
We also need a `LICENSE` file like https://github.com/mozilla/fxa-content-server/blob/master/LICENSE